File: FrizerskiSalon-API/FrizerskiSalon/Models/DTOs/Term.cs
namespace FrizerskiSalon.Models.DTOs
{
    public class Term
    {
        public long termID { get; set; }

        public long barberShopID { get; set; }

        public DateTime dateTime { get; set; }

        public TimeOnly startTime { get; set; }

        public TimeOnly endTime { get; set; }

        public long userId { get; set; }

        public long serviceID { get; set; }

        public bool busy { get; set; }

        public long workerID { get; set; }
    }
}

File: FrizerskiSalon-FRONT/src/models/Term.ts
import { Time } from "@angular/common";

export interface Term {
    termID: number;
    barberShopID: number;
    dateTime: Date;
    startTime: string;
    endTime: string;
    userID: number;
    serviceID: number;
    busy: boolean;
    workerID: number;
}

export interface TermSimple {
    booked: boolean;
    startTime: Date; // datum i vreme pocetka
    duration: number; //trajanje termina u minutima
}
